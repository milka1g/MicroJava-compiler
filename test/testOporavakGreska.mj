//testovi za oporavak od greske za definiciju globalne promenljive i dodelu vrednosti

program test301
	//primer za do zareza
	int a,,b;
	//primer za citavu definiciju jer ne valja cela
	int ,a,c,;
	//primer za globalnu prom koja se init, a to ne sme
	int m = 5;
{
	void main()	
	{
		bodovi = 0;
		bodovi++;

		//primer za dodelu vrednosti kad je izraz neispravan, Dodela vrednosti 
		bodovi = -8+;
		print(bodovi);
		print(bodovi);
		//primer za dodelu vrednosti kad izraz ni ne postoji, full Dodela vrednosti 
		bt = true++1;
	}
}