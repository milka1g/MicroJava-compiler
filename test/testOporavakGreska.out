Obrada ulaznog fajla test\program.mj
#2 program
#49 test301
#49 int
#49 a
#8 ,
#8 ,
#49 b
#5 ;
#49 int
#8 ,
#49 a
#8 ,
#49 c
#8 ,
#5 ;
#49 int
#49 m
#46 =
#47 5
#5 ;
#3 {
#25 void
#49 main
#6 (
#7 )
#3 {
#49 bodovi
#46 =
#47 0
#5 ;
#49 bodovi
#15 ++
#5 ;
#49 bodovi
#46 =
#30 -
#47 8
#31 +
#5 ;
#9 print
#6 (
#49 bodovi
#7 )
#5 ;
#9 print
#6 (
#49 bodovi
#7 )
#5 ;
#49 bt
#46 =
#50 true
#15 ++
#47 1
#5 ;
#4 }
#4 }
